app/views/shared/item_deleted.html.erb

```diff
@@ -0,0 +1,14 @@
+<div class="item-deleted-show">
+  <div id="<%= dom_id(@resource) %>" class="row">
+    <div class="small-12 medium-9 column">
+      <div class="callout primary">
+        <%= t("shared.deleted_item",
+              resource: link_to(t("#{@resource.class.to_s.downcase.pluralize}.show.#{@resource.class.to_s.downcase.pluralize}"), proposals_path, class: "probe-message"),
+              home_page: link_to(t("shared.homepage"), root_path, class: "probe-message"),
+              org: setting['org_name']).html_safe %>
+      </div>
+    <aside class="small-12 medium-3 column">
```

Comment:
Hi @ortegacmanuel! We have to avoid empty HTML, could you  please delete this empty `<aside>` 🙏 

Thanks!

Reply:
Done! I also added a missing `</div>`. Doing this change I noticed that the file's name `item_deleted` does not start with an `_`. Should I rename it in order to follow that name formate. All other files's names in shared starts with an `_` 

If is needed to rename the file name, just tell me ;)